AWSPinpoint/AWSPinpointEndpointProfile.m

```diff
@@ -520,11 +526,19 @@ - (void)encodeWithCoder:(NSCoder *)encoder {
 
 @implementation AWSPinpointEndpointProfileUser
 
-- (NSString*) quotedString:(NSString*) input {
+- (instancetype) init {
+    if (self = [super init]) {
+        _userId = @"";
+        _userAttributes = [NSMutableDictionary dictionary];
+    }
+    return self;
+}
+
```

Comment:
I don't think we should be adding an init function here?  why do we need this?

Reply:
Please remove the init.  It is not needed, and since `AWSPinpointEndpointProfileUser` conforms to NSSecureCoding, adding this init actually breaks conformance with that protocol